lxd/container_lxc.go

```diff
@@ -1083,6 +1083,25 @@ func (c *containerLXC) Start() error {
 		return err
 	}
 
+	// If stateful, restore now
+	if stateful && shared.PathExists(c.StatePath()) {
```

Comment:
Do you want to warn or error if --stateful was requested but the state doesn't exist?